test/irb/test_load_path_prepend_option.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: false
+require 'test/unit'
+require 'irb'
+require 'irb/lc/error'
+
+module TestIRB
+  class TestLoadPathPrependOption < Test::Unit::TestCase
+    def test_valid_load_path
+      begin
+        path = File.expand_path('lib', Dir.pwd)
+        IRB.parse_opts(argv: ['-I', path])
```

Comment:
The current directory is not the source directory.
`./lib` directory may not exist.